### ISSUE TITLE
Build broken: coverage is failing on all PRs

### ISSUE THREAD

**Issue by jhugman** [mozilla/application-services#5136]
This isn't blocking any PRs being landed.

If no-one cares about it, should we remove it?
If people do care about it, can we fix it?

```
   Compiling atty v0.2.14
   Compiling time v0.1.44
error[E0658]: trait bounds other than `Sized` on const fn parameters are unstable
  --> /home/circleci/.cargo/registry/src/github.com-1ecc6299db9ec823/quick-xml-0.25.0/src/writer.rs:62:6
   |
62 | impl<W: Write> Writer<W> {
   |      ^
63 |     /// Creates a `Writer` from a generic writer.
64 |     pub const fn new(inner: W) -> Writer<W> {
   |     --------------------------------------- function declared as const here
   |
   = note: see issue #93706 <https://github.com/rust-lang/rust/issues/93706> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `quick-xml` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `grcov v0.8.12`, intermediate artifacts can be found at `/tmp/cargo-install44QRHG`

Caused by:
  build failed

Exited with code exit status 101

CircleCI received exit code 101

```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/SYNC-3323)

**Comment by tarikeshaq** [comment]
@bendk do you happen to know if this is the same error that was fixed yesterday or a new one 😅

**Comment by tarikeshaq** [comment]
heh from https://github.com/mozilla/application-services/pull/5137 looks like it fixed itself?

**Comment by bendk** [comment]
I'm pretty sure this was fixed by upgrading the Rust version (#5128)

**Comment by tarikeshaq** [comment]
Looks like this is no longer a problem as all recent PRs have it working
